Assets/GameAssets/Player/PlayerScripts/PlayerHealth.cs

```diff
@@ -43,8 +48,40 @@ public override void IncreaseHealth(float heal)
         base.IncreaseHealth(heal);
 
         PlayerManager.Instance.InvokePlayerHealthChangeEvent(GetHealthPercent(), _currentHealth);
+        CheckHeartBeat();
     }
 
+    private void CheckHeartBeat()
+    {
+        if (base._currentHealth <= _HeartBeatStart && base._currentHealth != _HeartBeatEnd && _heartBeatCoroutine == null)
+        {
+            StartCoroutine(HeartbeatLoop());
+        }
+        if (base._currentHealth == _HeartBeatEnd && _heartBeatCoroutine != null)
+        {
+            StopCoroutine(HeartbeatLoop());
+        }
+    }
+/// <summary>
+/// A coroutine that is mean't to loop the heartbeat sfx
+/// </summary>
+/// <returns>returns null to loop similar to update function</returns>
+    IEnumerator HeartbeatLoop()
```

Comment:
Indent the comment and mark this as private (Yeah I know it's naturally private if not specified, but I feel we should maintain that standard)

Reply:
^ specify private